aframe/src/networking.js

```diff
@@ -1,32 +1,61 @@
 import {log} from './logger';
+import axios from 'axios';
 import uuidv4 from 'uuid/v4';
 
 // Modify to test a local server
 // const API_BASE = 'http://localhost:3000/1.0';
 
 // TODO: Need to change the API base The Graph to fetch correct ad
-const API_BASE = null;
-const AD_ENDPOINT = API_BASE + '/ad';
+const AD_ENDPOINT = 'https://api.thegraph.com/subgraphs/name/zestymarket/zesty-market-rinkeby'
 
 // TODO: Metrics should go to something like Textile
 const METRICS_ENDPOINT = null;
 
 const sessionId = uuidv4();
 
-const fetchAd = async (auId) => {
-  log(`Ad unit ID: ${auId}`);
+const fetchAd = async (tokenGroup, publisher) => {
+  const currentTime = Math.floor(Date.now() / 1000);
 
-  return fetch(`${AD_ENDPOINT}?au_id=${auId}`, {
-    method: 'GET',
+  return axios.post(AD_ENDPOINT, {
+    query: `
```

Comment:
We should give this query a name and place it in a different file where we curate our queries.